manifests/quick-start/sso/overlays/argo-server-sa.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: argo-server
+  annotations:
+    # The rule is an expression used to determine if this service account should be used. 
+    # * `groups` - an array of the OIDC groups
+    # * `iss` - the issuer
+    # * `sub` - the subject (typically the username)
+    # Must evaluate to a boolean. 
+    # If you want an account to be the default to use, this rule can be "true".
+    workflows.argoproj.io/rbac-rule: "'authors' in groups"
```

Would love some pointers here to the language used for these expressions. My first thought looking here was "How do I link the SA with multiple groups?".

---

It's this: https://github.com/antonmedv/expr

But yes, we should provide some examples. 

You have:

- `sub`
- `iss`
- `groups` if you have configured them in sso: {groups: []}

---

It then probably makes sense to add something like this:
```suggestion
    # Details of the expression language are available in
    # https://github.com/antonmedv/expr/blob/master/docs/Language-Definition.md.
    workflows.argoproj.io/rbac-rule: "'authors' in groups"
```